VisualPinball.Unity/VisualPinball.Unity/Display/ScoreReelComponent.cs

```diff
@@ -23,6 +23,14 @@ namespace VisualPinball.Unity
 {
 	public class ScoreReelComponent : MonoBehaviour
 	{
+		public enum ScoreReelDirection
+		{
+			up, down
```

Comment:
I looked pretty quick (so possibly I missed), but all the enums in the project I saw always begin with a capital letter.

Reply:
Changed!